pkg/kv/kvprober/settings.go

```diff
@@ -134,3 +134,9 @@ var quarantineWriteInterval = settings.RegisterDurationSetting(
 		"kv.prober.write.timeout controls the max time kvprober can be blocked",
 	10*time.Second,
 	settings.PositiveDuration)
+
+var tracingEnabled = settings.RegisterBoolSetting(
+	settings.ApplicationLevel,
+	"kv.prober.tracing.enabled",
+	"whether the KV prober should collect traces",
```

Comment:
"whether the KV prober should collect traces" -> "whether the KV prober should collect traces, in order to log info about leaseholders of ranges probed"

Reply:
Changed, thank you!